docs/features/feature-guides/compliance-forms/create-a-form.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Create a Form
+---
+
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
+
+# Create a DataHub Compliance Form
+<FeatureAvailability/>
+
+This guide will walk you through creating and assigning Compliance Forms, including:
+
+1. Defining your Compliance Form
+2. Creating Questions to be completed by assignees
+3. Selecting the in-scope Assets for the Compliance Form
+4. Assigning Forms to specific Users
+5. Publishing your Form
+
+:::note
+Creating and managing Compliance Forms via the UI is only available in DataHub Cloud. If you are deployed with DataHub Core, please see the [Compliance Forms API Guide](../../../api/tutorials/forms.md).
+:::
+
+## Create and Assign a Compliance Form from the DataHub Cloud UI
+
+### Prerequisites
+
+In order to create or edit a Compliance Form, your DataHub User must have the **Manage Compliance Forms** permission.
+
+### Step 1: Define your Compliance Form
+
+From the navigation bar, head to **Govern** > **Compliance Forms**. Click **+ Create** to start building your Form.
+
+<p align="center">
+  <img 
+       width="90%"  
+       src="https://raw.githubusercontent.com/datahub-project/static-assets/refs/heads/main/imgs/compliance_forms/management/list-compliance-forms-before.png"
+       alt="View of all Compliance Forms"/>
+</p>
+
+First up, provide the following details:
+
+1. **Name:** Give your Compliance Form a unique name.
+2. **Description:** Describe the purpose of the Form to help your users understand the purpose of the exercise.
+3. **Type:** Determine the collection type of the Form:
+    - **Verification:** Collect required information and require final verification to complete the Form.
+    - **Completion:** Collect required information; final verification is not required.
```

Comment:
Let's put "completion" first, because it is the simpler option and we opt for the principle of _progressive disclosure_ or _incremental complexity_ when we build things